locations/spiders/roche_bobois.py

```
@@ -0,0 +1,42 @@
+        try:
+            oh = OpeningHours()
+            for rule in ld_data["openingHours"]:
+                day, times = rule.split(" ", 1)
+                if times == "Closed":
+                    oh.set_closed(day)
+                else:
+                    for time in times.split(" / "):
+                        oh.add_range(day, *time.split(" - "), time_format="%I:%M %p")
+            item["opening_hours"] = oh
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)